modules/services/pass-secret-service.nix

```
@@ -28,10 +28,11 @@ in
       defaultText = "$HOME/.password-store";
       example = "/home/user/.local/share/password-store";
       description = ''
-        Absolute path to password store. Defaults to
-        {file}`$HOME/.password-store` if the
-        {option}`programs.password-store` module is not enabled, and
-        {option}`programs.password-store.settings.PASSWORD_STORE_DIR` if it is.
+        Absolute path to password store, default upstream value of which is
```

This makes things a little more convoluted language-wise... Perhaps this instead?

```suggestion
        Absolute path to password store, which has a default upstream value of
```

---

Thanks for the careful feedback @cyntheticfox, your version is definitely
clearer here.

At first I only tweaked the existing description since my focus wasn't this
module. My intention was to merely update it with the changes to
`password-store`.

But now that I look at it, I'm doubting if the default path of a dependency
should be explained on this module. Wouldn't it be better to avoid the part
after comma? It sounds as if we set such a default if the word "upstream"
doesn't capture enough attention of the reader.

Considering the other feedback I would suggest this at the end

```txt
Absolute path to the password store. If the
{option}`programs.password-store` module is enabled, the
{option}`programs.password-store.settings.PASSWORD_STORE_DIR` option
will be checked, if found it will be inherited as the default.
```

---

That's probably fine -- I'm just not entirely certain without sitting at a terminal (I'm trying to do this on a phone at work 😅) how the Systemd Environment inheritance and cleaning will interact with it. I'm pretty sure I only set it this way to make sure it copied the behaviour of the original module, so since this is a breaking change anyways it seems fine to just remove it -- do all the breaking at the same time